package.json

```diff
@@ -119,6 +119,7 @@
     "gzip-size": "^3.0.0",
     "image-size": "^0.5.1",
     "jasmine-core": "^2.4.1",
+    "jsdom": "^9.5.0",
```

Comment:
Why not use the latest version [`jsdom@11`](https://www.npmjs.com/package/jsdom)?

Reply:
Because `9.5.0` is what we have in streambed, and it looked like it was going to take me more than 5 minutes to figure out the new API 🙈 

But you're right, not exactly a good reason - I'll try to upgrade it.